bin/bank/pycbc_brute_bank

```diff
@@ -53,6 +53,8 @@ parser.add_argument('--approximant',  required=False,
 parser.add_argument('--minimal-match', default=0.97, type=float)
 parser.add_argument('--buffer-length', default=2, type=float,
     help='size of waveform buffer in seconds')
+parser.add_argument('--buffer-high-pass-length', default=None, type=float,
```

Comment:
The name / help might be clearer. Maybe --full-resolution-buffer-length or maybe you have a better idea? Suggestions?

Reply:
--full-resolution-buffer-length sounds good! If it feels too lengthy, perhaps --full-res-buffer-length could be a more concise alternative.